README.md

```diff
@@ -1,5 +1,21 @@
+[Dynatrace](https://www.dynatrace.com/integrations/opentelemetry) supports native
+OpenTelemetry protocol (OTLP) ingest for traces, metrics and logs.
+All signals can be sent directly to Dynatrace via **OTLP protobuf over HTTP**
+using the built-in OTLP/HTTP Exporter available in the OpenTelemetry Java SDK.
+More information on configuring your Java applications to use the OTLP exporter can be found in the
+[Dynatrace documentation](https://www.dynatrace.com/support/help/shortlink/otel-wt-java#tabgroup--dynatrace-docs--otlp-export).
+
 # Dynatrace OpenTelemetry Metrics Exporter for Java
```

Comment:
The other option than the table would be to show the deprecation in the title for example like this:
# Dynatrace OpenTelemetry Metrics Exporter for Java (deprecated)

Reply:
I added the badge instead: ![Static Badge](https://img.shields.io/badge/status-deprecated-orange).